<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapMenu</class>
 <widget class="QDialog" name="MapMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bludiště 2014</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>30</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideLeft</enum>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="createTab">
      <attribute name="title">
       <string>Založit hru</string>
      </attribute>
      <widget class="QListWidget" name="mapList">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>250</width>
         <height>220</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Dostupné mapy</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="delayBox">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>100</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="suffix">
        <string>s</string>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>5.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
      <widget class="QSpinBox" name="playersBox">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>130</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>106</y>
         <width>59</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Prodleva</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>285</x>
         <y>134</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max. hráčů</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nameTxt">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>160</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>nova hra</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>163</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Název hry</string>
       </property>
      </widget>
      <widget class="QPushButton" name="createBtn">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>210</y>
         <width>130</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>Založit</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="refreshMapsBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>250</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>obnovit seznam</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>74</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>61</y>
         <width>59</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="mapIdTxt">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>70</y>
         <width>58</width>
         <height>23</height>
        </rect>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="joinTab">
      <attribute name="title">
       <string>Připojit se</string>
      </attribute>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Rozehrané hry</string>
       </property>
      </widget>
      <widget class="QListWidget" name="gameList">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>250</width>
         <height>220</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="joinBtn">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>210</y>
         <width>130</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>Připojit se</string>
       </property>
      </widget>
      <widget class="QPushButton" name="refreshGamesBtn">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>280</y>
         <width>250</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>obnovit seznam</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>285</x>
         <y>134</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Max. hráčů</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="showMaxPlayersBox">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>130</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>317</x>
         <y>106</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Hráčů</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="showPlayersBox">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>100</y>
         <width>61</width>
         <height>24</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="gameIdTxt">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>70</y>
         <width>58</width>
         <height>23</height>
        </rect>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>74</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
